src/bokeh/events.py

```diff
@@ -316,7 +316,7 @@ class ButtonClick(ModelEvent):
 
     def __init__(self, model: AbstractButton | None) -> None:
         from .models.widgets import AbstractButton, ToggleButtonGroup
-        if model is not None and not isinstance(model, AbstractButton | ToggleButtonGroup):
+        if model is not None and not isinstance((model, AbstractButton , ToggleButtonGroup)):
```

Comment:
This change is incorrect 
```suggestion
        if model is not None and not isinstance(model, (AbstractButton, ToggleButtonGroup)):
```

Reply:
@meh-21 Please double check there are not any other erroneous changes like this one (the CI test output is where to look)